### PR TITLE
Add swagger

### LINKED ISSUE #20: Swagger
shall we add generated swagger files in `api-spec/openapi/swagger`?

### PR DESCRIPTION
**Pull request by altafan**
Closes #20 

Please @tiero review.

### CODE REVIEW

**Comment by tiero** [comment]
There is inconsistency between taxi-protobuf & tdex-protobuf.

I am in favor of having THIS folder structure, but we changed in tdex-protobuf, so best to be in line in a way or another?

File: package.json [hunk 1]
```diff
@@ -12,5 +12,8 @@
     "grpc-tools": "^1.10.0",
     "grpc_tools_node_protoc_ts": "^5.1.0",
     "protoc-gen-grpc-web": "^1.2.1"
+  },
```
Comment (tiero) [inline package.json]: since we use buf, we do not need anymore pkg.json?

Comment (altafan) [reply package.json]: yeah I was thinking the same, i just wanted confirmation before removing

**Comment by altafan** [comment]
> There is inconsistency between taxi-protobuf & tdex-protobuf.
> 
> I am in favor of having THIS folder structure, but we changed in tdex-protobuf, so best to be in line in a way or another?

Agreed. We are going to fix tdex-protobuf folder structure.

**Comment by tiero** [comment]
lets remove the pkg.json entirely?

**Comment by altafan** [comment]
@tiero done.

**Pull request merged**